pkg/common/policy/policy.go

```diff
@@ -0,0 +1,170 @@
+package policy
```

Comment:
This package seems fairly server-centric, i.e. in the inputs and outputs and tests that are performed. I wonder if some of it belongs in a server policy package. Since we don't have any agent rego plans yet, maybe this should all be defined under server and then the common code refactored out later if/when needed....